experimental/sql/tools/rel_opt.py

```diff
@@ -39,6 +39,7 @@ def register_all_frontends(self):
     def parse_ibis(f: IOBase):
       import ibis
       import pandas as pd
+      import numpy as np
```

Comment:
This is needed for the `np.int64(0)` in the queries, since o/w, ibis seems to parse integers as i8.